### DISCUSSION TITLE
Instructions for windows install unclear

### DISCUSSION THREAD

**Discussion by awesomec26** [pystardust/ani-cli#319] [Q&A] [answered]
I understand everything else to install on windows but the "Add vlc to Windows Env PATH like so: C:\Program Files\VideoLAN\VLC" instruction, what do I do?

**Comment by JRCdev** [comment, marked as answer]
Windows search "Edit the system environment variables" -> "Advanced" tab -> "Environment Variables..." button -> "Path" user variable + "Edit" button -> "New" and then enter your VLC path, then "OK"
